core/src/main/java/com/graphhopper/routing/InstructionsOutgoingEdges.java

```diff
@@ -139,7 +139,6 @@ public boolean outgoingEdgesAreSlowerByFactor(double factor) {
         for (EdgeIteratorState edge : visibleAlternativeTurns) {
             tmpSpeed = getSpeed(edge);
             if (tmpSpeed < 1) {
-                // This might happen for the DataFlagEncoder, might create unnecessary turn instructions
                 return false;
```

Comment:
Do you know if this is still necessary?

Reply:
No, unfortunately I do not know. Maybe remove it if no tests fail due to removing?

Reply:
@boldtrn maybe?

Reply:
The tests are green also after removing it. So let's remove it and put it back if someone can show a case where this is important.

Reply:
Not sure if this case can still happen, probably not, so makes sense to remove it :)

Reply:
Ok thanks for taking a look :+1: